### PR TITLE
feat(NRQL intro): adding charts and dashboards doc (68350)

### PR DESCRIPTION
**Pull request by jeff-colucci**
Adding the base doc for the new charts and dashboards part of the NRQL intro

### CODE REVIEW

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<img
+  title="nrql-dashboard-example.png"
+  alt="A screenshot example of a dashboard"
+  src={dashboardsDashboardUi}
+/>
+<figcaption>
+  A dashboard representing data in pie and bar chart formats.
+</figcaption>
+
+While charts are great for reporting on individual pieces, dashboards are the best way to get a much broader view of your data: if charts are pieces to a puzzle, dashboards are the puzzle completed. Dashboards are highly configurable, and you can use them to show you as many or as few data streams as you like. You can:
```
Comment (rhetoric101) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:51]: I really like the puzzle metaphor!

I'm not sure you need the "as many or as few" when "as many" might suffice.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+* [Add charts](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#dash-create-chart) (using the query builder) or your own [text, images, or links](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#markdown) (using Markdown) to your dashboard
```
Comment (rhetoric101) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:53]: @jeff-colucci this was a bit tough for me to parse with the parenthetical comments. How about just making separate bullets?

* [Add charts](https://docswebsitedevelop-nrqlintrochartsdashboards.gatsbyjs.io/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#dash-create-chart) to your dashboard using the query builder
* Add your own text, images, or links](https://docswebsitedevelop-nrqlintrochartsdashboards.gatsbyjs.io/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#markdown) using Markdown

**Review by rhetoric101 — APPROVED** [review]
@jeff-colucci This is a well-written doc. I just inserted a few small suggestions. Still, it seems like this doc overlaps quite a bit with the main dashboards and charts docs. While there are some NRQL-specific comments, it seems pretty similar to the other docs. I wonder if you need to get this more focused on NRQL, or maybe just add some clarifying text in the doc to explain how this relates to those other docs?

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Charts and dashboards: visualizing your data"
+tags:
+  - Visualize your data
+  - 'NRQL: New Relic query language'
+  - Get started
+translate:
```
[the remark this answers is not in the document]
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:7]: Thanks for letting me know, I'll check in on this

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  - jp
+  - kr
+metaDescription: "Learn the basics of charts and dashboards, and how they relate to NRQL."
+redirects:
+---
+import dashboardsDashboardUi from 'images/dashboards_screenshot-crop_dashboard-ui.png'
+
+import chartsChartUi from 'images/queries-nrql_screenshot-crop_dashboard-chart-color.png'
+
+Knowing how to query your data is important for finding the information you need, but knowing how to visualize your data is important for making that information understandable for you and for your organization. Charts and dashboards are the key to data visualization, giving you multiple options that enable you to create visually appealing and easily understandable data streams. There are some key things to know about these features to see how they relate to your NRQL queries, and understanding them helps you make the most of your data.
```
[the remark this answers is not in the document]
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx]: Made some changes based on this feedback, and thought about ways we could make the charts and NRQL connection more concrete for new users. Trying to solidify the flow as NRQL query > chart > dashboard.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+<img
+  title="nrql-chart-example.png"
+  alt="A screenshot example of a chart"
+  src={chartsChartUi}
+/>
+<figcaption>
+  This is a chart created by a NRQL query showing the total number of transactions for each reported application across a period of 1 hour.
+</figcaption>
+
+Configuring a chart to represent your data is often the last step of creating a NRQL query. New Relic offers [multiple options](/docs/query-your-data/explore-query-data/use-charts/chart-types) to help you find the perfect way to visualize your data, and you can do many things with your charts once you've made them. You can:
```
[the remark this answers is not in the document]
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx]: You are correct, thank you for letting me know.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx (hunk 1 above)
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx]: Thanks! I tweaked the sentence slightly based on your feedback as well.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx (hunk 2 above)
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx]: Good call, changes have been made!

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+import dashboardsDashboardUi from 'images/dashboards_screenshot-crop_dashboard-ui.png'
+
+import chartsChartUi from 'images/queries-nrql_screenshot-crop_dashboard-chart-color.png'
+
+Knowing how to query your data is important for finding the information you need, but knowing how to visualize your data is important for making that information understandable for you and for your organization. Charts and dashboards are the key to data visualization, giving you multiple options that enable you to create visually appealing and easily understandable data streams. There are some key things to know about these features to see how they relate to your NRQL queries, and understanding them helps you make the most of your data.
+
+## Charts [#charts]
+
+<img
+  title="nrql-chart-example.png"
```
[the remark this answers is not in the document]
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx]: Will do!

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+import dashboardsDashboardUi from 'images/dashboards_screenshot-crop_dashboard-ui.png'
+
+import chartsChartUi from 'images/queries-nrql_screenshot-crop_dashboard-chart-color.png'
+
+Queries help you find the information you need, and your charts and graphs based on those queries make the data useful to your organization. Charts and dashboards are the key to data visualization, giving you multiple options that enable you to create visually appealing and easily understandable data streams. You can make the most of your data by understanding the ways that NRQL queries lead in to charts and dashboards.
+
+## Charts [#charts]
+
+<img
+  title="Chart screenshot.png"
```
Comment (jeff-colucci) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:22]: Make alt text into title

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  alt="A screenshot example of a chart"
+  src={chartsChartUi}
+/>
+<figcaption>
+  This is a chart created by a NRQL query showing the total number of transactions for each reported application across a period of 1 hour.
+</figcaption>
+
+Configuring a chart to represent your data is the last step of creating a NRQL query. New Relic offers [multiple options](/docs/query-your-data/explore-query-data/use-charts/chart-types) to help you find the perfect way to visualize your data, and you can do many things with your charts once you've made them. After you write your NRQL query, you can use charts to:
```
Comment (jeff-colucci) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:30]: Flesh out the process for creating a chart after NRQL. Ie: After writing the query and then you need to create a chart and save it.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx (hunk 7 above)
Comment (jeff-colucci) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:22]: change screenshot to show chart creation interface

**Review by rhetoric101 — APPROVED** [review]
@jeff-colucci thanks for adding this new overview doc! It looks good!

**Pull request merged**